src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx

```diff
@@ -218,7 +211,7 @@ export default function DownloadDialog({
   }
 
   useEffect(() => {
-    const getIinstallInfo = async () => {
+    const getIinstInfo = async () => {
```

Comment:
Typo ?

```suggestion
    const getInstallInfo = async () => {
```

Reply:
Not a typo, the problem is that we already have a getInstallInfo from import.
So in lack of a better name, I intentionally changed this one.